modules/admin-portal/pages/monitoring/log-viewer.adoc

```diff
@@ -57,4 +57,33 @@ image:single-search-result.png[image]
 
 Clicking on a single search result will show the log content in the display panel and highlight the matching log line.
 
-image:line-highlight-log (1).png[image]
\ No newline at end of file
+image:line-highlight-log (1).png[image]
+
+== Read-Only Access to Logs
+
+To streamline the debugging and optimization process, *read-only access to logs* has been introduced for application developers. This feature allows developers to independently view and monitor logs through *GraphStudio* or the *TigerGraph Cloud Portal* without requiring sysadmin intervention.
+
+=== Privileges
+
+==== APP_ACCESS_LOG Privilege
+The `APP_ACCESS_LOG` privilege provides *global access* to logs for authorized users.
+* *Scope:* Global
+* *Default Roles:* Assigned to `superuser` and `globaldesigner` roles by default.
+* *Purpose:* Grants access to view logs across all environments (on-prem and cloud).
+
+==== READ_LOG Privilege
+The `READ_LOG` privilege ensures *read-only access* to logs, preventing modifications while enabling secure monitoring and debugging.
```

Comment:
```suggestion
The `READ_LOG` privilege grants *read-only access* to logs, preventing modifications while enabling monitoring and debugging.
```
"Ensures": "grant" is the standard verb for privilege.  "Ensures" emphasizes "guarantees". That is, if you didn't ensure, than this thing might or might not happen. It introduces the idea of uncertainty, but uncertainty is not relevant to this discussion.

"Secure" : how?
I feel this sentence, and other parts of this proposal, are influenced by marketing (saying things that sound impressive) and not focusing on the facts.  If a feature is inherently beneficial, then (1) just say what is done, in comparison to other features or if you didn't hvae it, and (2) then explain it's benefits.

 Marketing writing jumps to the benefits (some of which are exaggerated) without a clear explanation of what the feature does or now it works.